src/services/pocket-relayer.ts

```diff
@@ -235,82 +206,86 @@ export class PocketRelayer {
         });
       }
     }
-    // Exhausted relay attempts; use fallback
+    // Exhausted network relay attempts; use fallback
     if (fallbackAvailable) {
       let relayStart = process.hrtime();
+      let axiosConfig = {};
 
-      const fallbackChoice = new HttpRpcProvider(this.fallbacks[Math.floor(Math.random() * this.fallbacks.length)]);
-      const fallbackPayload : FallbackPayload = {data: rawData.toString(), method: httpMethod, path: relayPath,  headers: null};
-      const fallbackMeta: FallbackMeta = {block_height: 0};
-      const fallbackProof: FallbackProof = {blockchain: blockchain};
-      const fallbackRelay: FallbackRelay = {
-        payload: fallbackPayload,
-        meta: fallbackMeta,
-        proof: fallbackProof,
-      };
-
-      const fallbackResponse = await fallbackChoice.send(
-        '/v1/client/relay',
-        JSON.stringify(fallbackRelay),
-        1200000,
-        false,
-      );
+      // Add relay path to URL
+      const altruistURL = (relayPath === undefined || relayPath === "") ? this.altruists[blockchain] : this.altruists[blockchain] + '/' + relayPath;
 
-      if (this.checkDebug) {
-        logger.log('debug', JSON.stringify(fallbackChoice), {
-          requestID: requestID,
-          relayType: 'FALLBACK',
-          typeID: application.id,
-          serviceNode: 'fallback:' + fallbackChoice.baseURL,
-        });
-        logger.log('debug', JSON.stringify(fallbackRelay), {
-          requestID: requestID,
-          relayType: 'FALLBACK',
-          typeID: application.id,
-          serviceNode: 'fallback:' + fallbackChoice.baseURL,
-        });
-        logger.log('debug', JSON.stringify(fallbackResponse), {
-          requestID: requestID,
-          relayType: 'FALLBACK',
-          typeID: application.id,
-          serviceNode: 'fallback:' + fallbackChoice.baseURL,
-        });
+      // Remove user/pass from the altruist URL
+      const redactedAltruistURL = String(this.altruists[blockchain])!.replace(/[\w]*:\/\/[^\/]*@/g, '');
+
+      if (httpMethod === 'POST') {
+        axiosConfig = {
+          method: 'POST',
+          url: altruistURL,
+          data: rawData.toString(),
+          headers: {'Content-Type': 'application/json'}
+        };
+      } else {
+        axiosConfig = {
+          method: httpMethod,
+          url: altruistURL,
+          data: rawData.toString()
+        };
       }
-      if (!(fallbackResponse instanceof RpcError)) {
-        const responseParsed = JSON.parse(fallbackResponse);
+      try {
+        const fallbackResponse = await axios(axiosConfig);
+
+        if (this.checkDebug) {
+          logger.log('debug', JSON.stringify(fallbackResponse.data), {
+            requestID: requestID,
+            relayType: 'FALLBACK',
+            typeID: application.id,
+            serviceNode: 'fallback:' + redactedAltruistURL,
```

Comment:
Same thing here and in the other places below:
```suggestion
            serviceNode: `fallback:${redactedAltruistURL}`,
```

Reply:
tbh I do this everywhere in the code. It will have to be prettified later. Hopefully automatically.